.eslintrc.js

```diff
@@ -450,15 +428,7 @@ module.exports = {
             forceNode: false,
             rootPackageName: 'kibana',
             kibanaPath: '.',
-            pluginMap: readdirSync(resolve(__dirname, 'x-pack/legacy/plugins')).reduce(
-              (acc, name) => {
-                if (!name.startsWith('_')) {
-                  acc[name] = `x-pack/legacy/plugins/${name}`;
-                }
-                return acc;
-              },
-              {}
-            ),
+            pluginMap: {},
```

Comment:
I'm not sure this rule makes any sense without any legacy plugin. I changed `pluginMap` to `{}`, but should I just remove the whole rule instead @elastic/kibana-operations ?

Reply:
I think that the custom eslint-import-resolver-kibana settings for x-pack are only required for the legacy import syntax support for things like `ui/*` and `plugins/*`. I suspect that x-pack can use the root settings and we can get rid of these settings.

Reply:
Actually still necessary. See https://github.com/elastic/kibana/pull/77599/checks?check_run_id=1147976384